resources/lib/database/db_shared.py

```diff
@@ -23,19 +23,6 @@ def __init__(self):
             else:
                 super(NFSharedDatabase, self).__init__(db_utils.SHARED_DB_FILENAME)
 
-        def get_value(self, key, default_value=None, table=db_utils.TABLE_SHARED_APP_CONF,
```

Comment:
Please do not delete those overrides or it will try to query the app_config table instead of the shared_app_config one, resulting into errors.

Reply:
Will revert. I didn't notice the default value of tables were being set. Thanks for noticing!